src/EnergyPlus/CMakeLists.txt

```diff
@@ -968,10 +972,16 @@ if(LINK_WITH_PYTHON)
     TARGET energyplusapi
     POST_BUILD # TODO: I don't think we want to quote the generator expression
     COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonCopyStandardLib.py" "$<TARGET_FILE:energyplusapi>" "python_lib"
-    COMMAND ${CMAKE_COMMAND} -E env --unset=PIP_REQUIRE_VIRTUALENV
-            ${Python_EXECUTABLE} -m pip install --target="$<TARGET_FILE_DIR:energyplusapi>/python_lib" --upgrade energyplus-launch==3.7.2
-    COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonFixUpTclTk.py" "$<TARGET_FILE_DIR:energyplusapi>/python_lib"
   )
+  if (PYTHON_CLI)
```

Comment:
Split the PythonCopyStandardLibrary.py and the Pip Install with Tcl Fixup stuff into two separate commands.  This way we can protect the second one within an IF (PYTHON_CLI) condition.

Reply:
If you had enabled LINK_WITH_PYTHON and PYTHON_CLI, and you disable LINK_WITH_PYTHON, this block would still be executed.

Reply:
Oh, I thought this whole section was wrapped inside a LINK_WITH_PYTHON check.

Reply:
This whole section is indeed wrapped inside a LINK_WITH_PYTHON condition up at line 918.  This one is fine I think.  But I did find a spot just below where I am creating the shortcut lnk file to EPLaunch that needed extra protection.  If you confirm that it definitely needs change here, let me know.